testgrid/config.yaml

```diff
@@ -5053,15 +5058,6 @@ dashboards:
   - name: gke-gci-1.12-gci-1.11-downgrade-cluster-parallel
     test_group_name: ci-kubernetes-e2e-gke-gci-stable1-gci-stable2-downgrade-cluster-parallel
 
-- name: sig-release-misc
-  dashboard_tab:
-  - name: cross-build
-    test_group_name: ci-kubernetes-cross-build
-  - name: kops-build
-    test_group_name: ci-kops-build
```

Comment:
you probably want to put it somewhere under kops or sig-aws?

Reply:
Why?

SIG-cluster-lifecycle has 11 Kops tests already, but they *don't* include this one.  If they don't think it's worth looking at, why would I put it on their board?

Reply:
cc @justinsb

Reply:
what else do I need to do to delete the job?